tests/data/test_summaries.py

```diff
@@ -48,16 +52,59 @@ def expected_score_target_summary_cuts(res):
 
 
 class Test_Summaries:
-    def test_default_summaries(self, prediction_data, expected_default_summary):
+    @patch.object(seismogram, "Seismogram", return_value=Mock())
+    def test_default_summaries(self, mock_seismo, prediction_data, expected_default_summary):
+        fake_seismo = mock_seismo()
+        fake_seismo.output = "Score"
+        fake_seismo.target = "Target"
+        fake_seismo.event_aggregation_method = lambda x: "max"
         actual = undertest.default_cohort_summaries(prediction_data, "Has_ECG", [1, 2, 3, 4, 5], "ID")
         pd.testing.assert_frame_equal(actual, expected_default_summary, check_names=False)
 
+    @patch.object(seismogram, "Seismogram", return_value=Mock())
     def test_score_target_summaries(
-        self, prediction_data, expected_score_target_summary, expected_score_target_summary_cuts
+        self, mock_seismo, prediction_data, expected_score_target_summary, expected_score_target_summary_cuts
     ):
+        fake_seismo = mock_seismo()
+        fake_seismo.output = "Score"
+        fake_seismo.target = "Target"
+        fake_seismo.event_aggregation_method = lambda x: "max"
         groupby_groups = ["Has_ECG", expected_score_target_summary_cuts]
         grab_groups = ["Has_ECG", "Score"]
         pd.testing.assert_frame_equal(
             undertest.score_target_cohort_summaries(prediction_data, groupby_groups, grab_groups, "ID"),
             expected_score_target_summary,
         )
+
+    @patch.object(seismogram, "Seismogram", return_value=Mock())
+    @pytest.mark.parametrize("aggregation_method", ["min", "max", "first", "last"])
+    def test_event_score_match_score_target_summaries(
+        self, mock_seismo, aggregation_method, prediction_data, expected_score_target_summary_cuts
+    ):
+        fake_seismo = mock_seismo()
+        fake_seismo.output = "Score"
+        fake_seismo.target = "Target"
+        fake_seismo.event_aggregation_method = lambda x: aggregation_method
+        ref_event = "Target"
+        df_aggregated = event_score(prediction_data, ["ID"], "Score", ref_event, aggregation_method)
+        bins = [0, 0.5, 1.0]
+        labels = ["(0,0.5]", "(0.5,1]"]
+
+        # Create a new column for binned scores
+        df_aggregated["Score_Binned"] = pd.cut(
+            df_aggregated["Score"], bins=bins, labels=labels, include_lowest=False, right=True
+        )
+
+        # Group by the desired columns and count
+        entities_event_score = (
+            df_aggregated.groupby(["Has_ECG", "Target_Value", "Score_Binned"])
```

Comment:
I'm seeing a warning from this line in my pyenv, needs an explicit set of `observed`
```suggestion
            df_aggregated.groupby(["Has_ECG", "Target_Value", "Score_Binned"], observed=False)
```